### ISSUE TITLE
How we talk about Apostrophe

### ISSUE THREAD

**Issue by boutell** [punkave/best-practices#34]
A modest proposal...

When possible, we talk about Apostrophe internally by talking about it externally:

* By asking questions in Stack Overflow and tagging them `apostrophe-cms` so the community benefits.
* By reporting bugs and feature requests via github issues on the `apostrophe` repo.
* By contributing to the `apostrophe-documentation` repo.

All of our communications in these forums should be friendly and appropriate and inviting to the larger community.

A face to face conversation is obviously helpful and desirable too, and some discussions of the overall direction of Apostrophe are best had in person.

We should retire our apostrophe slack because nobody's invited and it has no memory. A single Apostrophe channel in our regular slack is fine and would avoid scattering the conversation, but it's better to use github and stack overflow if you can.

Needless to say, we always use pull requests when contributing to Apostrophe.

Thoughts?

**Comment by agilbert** [comment]
👍

**Comment by austinstarin** [comment]
Thanks for writing this up Tom! Sounds good on my end.

**Comment by agilbert** [comment]
Any other thoughts from folks on this? What are the actionable steps to "implement" this best practice?

**Comment by bgantick** [comment]
This all makes sense to me. I think Tom has outlined the actionable steps nicely - following the bullet points and archiving the Apostrophe slack are a logical next move.

**Comment by agilbert** [comment]
Good point. If we're all using a single #apostrophe channel in the P'unk Slack account, we can build ways to reinforce the bullet points into that channel.

**Comment by agilbert** [comment]
Pipe SO stuff in there.
Explore integrations into the docs and forum.
Maybe a bot yells at you or makes helpful suggestions...
